crates/rolldown/tests/esbuild/ts/ts_import_in_node_modules_name_collision_with_css/_config.json

```diff
@@ -6,5 +6,7 @@
         "import": "entry.ts"
       }
     ]
-  }
+  },
+  // hyf0: To pass this test, we need to finsih the builtin css support and add `.css` to the default resolve extensions.
+  "expectError": true
```

Comment:
I tracked the test https://github.com/rolldown/rolldown/issues/4271#issue-3013704710 here.